compiler/tests-full/stdlib.cma.expected.js

```diff
@@ -1823,8 +1815,8 @@
      var
       xs$0 =  /*<<seq.ml:133:12>>*/ match[2],
       x = match[1],
-      _an_ =  /*<<seq.ml:137:6>>*/ caml_call1(p, x);
-      /*<<seq.ml:137:9>>*/ if(! _an_) return _an_;
+      a =  /*<<seq.ml:137:6>>*/ caml_call1(p, x);
+      /*<<seq.ml:137:9>>*/ if(! a) return a;
```

Comment:
The rationale for this style of variable name `_an_` is that we could search for all occurrences of a variable. This becomes harder when all variable names are shorten.

Reply:
For debugging, you can always use `--debug stable_var` or we could implement another variation.

Reply:
Do you have a use case in mind where neither the new behavior nor the stable_var one work for you ?

Reply:
I've added a new debug `--debug var` that emits variables in the `_an_` style and used it in `stdlib.cma.expected.js`

Reply:
> Do you have a use case in mind where neither the new behavior nor the stable_var one work for you ?

I think this is mostly useful for constants. You want to easily find where a constant is used (or find the definition of a constant), but still get names from the source code when available.